src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,23 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String reverse;
```

Comment:
@olehedza try to declare the variable as close as possible to the first usage

Reply:
Done!